docs/about/contributing.mdx

```diff
@@ -138,6 +151,110 @@ We're deeply thankful for the backing from our sponsors, listed below, which has
     />
   </div>
   <div className="w-full sm:w-9/12 p-3">
-    Datadog is an observability and security platform for cloud applications, providing infrastructure monitoring, application performance monitoring, and log management. They sponsor us by granting free infrastructure monitoring and logging, aiding our development and operational efforts.
+    Datadog is an observability and security platform for cloud applications,
+    providing infrastructure monitoring, application performance monitoring, and
+    log management. They sponsor us by granting free infrastructure monitoring
+    and logging, aiding our development and operational efforts.
   </div>
 </div>
+
+## Contributing to the Meshtastic Blog
+
+### Purpose
+
+To foster growth and collaboration by sharing knowledge, updates, and innovations from the Meshtastic community.
+
+### Content Guidelines
+
+#### Acceptable Topics
+
+- Focus on Meshtastic-related themes, such as tutorials, use cases, development updates, and technology deep dives.
+- Posts must be your original work. Prior content originally published elsewhere cannot be submitted, but posts may be republished elsewhere after being posted on Meshtastic's blog.
+- Posts should not duplicate a previously posted blog unless it is to update or revise the original content due to changes.
+
+#### Unacceptable Topics
+
+- Avoid self-promotion or commercial content.
+- Promoting the sale of hardware, cases, etc., or driving traffic to other websites is unacceptable.
+- Affiliate links are not allowed unless prior approval has been granted by the Meshtastic project.
+
+#### Author Biography
+
+You may link to your personal work or website, but links to personal content can only be included in the appropriate author info, either in the front matter or the global authors file.
+
+- Acceptable links include URLs, email addresses, and social media sites.
```

Comment:
I think we need to mandate rel nofollow
https://www.webfx.com/blog/web-design/rel-nofollow/

Reply:
Perfect